lib/app/app.dart

```diff
@@ -50,6 +49,7 @@ import 'package:zc_desktop_flutter/ui/workspace_registeration_screen/create_work
     LazySingleton(classType: AuthService),
     LazySingleton(classType: ApiService),
     LazySingleton(classType: WorkspaceService),
+    Singleton(classType: ThemeService, resolveUsing: ThemeService.getInstance)
```

Comment:
Why register as a singleton, go through the package documentation and see if that's how it should be register.

Reply:
Just went through it, it was registered as singleton from the example, with a little google search I find out that it can be registered as LazySingleton too. 
Will make the requires edit